packages/neuron-ui/src/components/History/index.tsx

```diff
@@ -82,20 +81,24 @@ const History = (props: React.PropsWithoutRef<ContentProps & RouteComponentProps
   const onPageChange = useOnChangePage(search, pathname, history, queryFormatter)
   const onAddressRemove = useOnAddressRemove(search, pathname, history, queryFormatter)
 
-  if (loadings.transactions) {
-    return <div>Loading</div>
-  }
+  // if (loadings.transactions) {
```

Comment:
Please don't leave comment as unused code. Either delete them, or finish them when possible.

Reply:
removed in https://github.com/nervosnetwork/neuron/pull/312/commits/7f6980c25c9a7e80964a9a5f21bf3f417bdffc30